kafka/pdl-leesah-ingestor/src/main/kotlin/no/nav/amt/tiltak/kafka/pdl_leesah/LeesahIngestorImpl.kt

```diff
@@ -45,9 +48,25 @@ class LeesahIngestorImpl(
 			}
 		}
 
+		val navn = JsonUtils.fromJsonString<Navn>(genericRecord.get("navn").toString())
```

Comment:
Må ikke denne fortsatt oppgis som nullable? leesah topicen legger bare endringene på topicen så er ikke sikkert at det er et navn på recorden

Reply:
ah, takk! Tenkte ikke på det, skriver en test for det også